helm-charts/templates/data-plane/ratelimiter/ratelimiter-keystore-secret.yaml

```diff
@@ -0,0 +1,11 @@
ZFlvMC9BM2lTOFVPOG9nWHpQTjZNSlQ4OU9VZGRrYnhFeXZXRzZTbFByU28KZ1RNSFBQUDRtd1BQdWdYY1R2N0N6QzNydnFXbkoyQTc3bitvL0pmZFdBaTdlcVN0Z2xGbjhEdXdBZDVFaysrYgo1ZVdINEtuWlQzN01HZFRDZmZVNG54S2d1RXZocFY3L01KS292QVIybzFwMmhZVXpZWktnRGszZGh5NCtITFpvCnFkS1JHSGdFZmgvdUhoVU0vTStKYUNTdXl5U2Vub2kzRXR4NTI3OGpUWjQrT3BrY3NGRmRaQmZJUkkyTlp0Y3IKNkZvRGxYWU1SM2ZQWjgwcS90V2tOaWxYenMyQU1iWHI1S2FZMnlQSy9QVkhnK1ZWek9WSzVaMHgweCtnVnNQdgptUEpiUFhTZzdVV1pSN3VTai9qN3RHc2tpV0pyd296KzZNOFl0YXVOQnZXMzA5RmxQQTJkR2xIUXA0dDlobkErCjd3Q0RtL2JuODRLbk9JOXRsN1krUEt1K3RvdGZKa3RXY2NPNlJDMlVwM1BrN0UzeWEvZTcyUVFZajdDY0lySEoKa3o2OUJqVG9QU1A5WktPRmtFcVJBb0lCQVFEeHBlQlJqQjkwZm94RzBTM2g0ZElrdFVPTTRRYUxia2ZIT2JCUgpnL1hVazBMWGdZMFJTNU11VnVIL1Nxb2thUW8zdXVDTmpQUzRkallack9SMmd2eDJ2NHVGRDlBNHcyWituREk0CjJibnZuTkVjNW9KVDlTTURXVm9kY2lMVXlBdWx1blEyekxrcHZzZk5jZjNNVWtkMUNncGhZc2dKM3k0L2ZhaEwKNkg3TldJY2tCSjdXck1SOGdXRURKa0wzNkNmMzEvTmNUK3ZMVk5zbURuMnpnbk1yeUcxZUh1SWMxR1BDbnRxUwpNVmpBeHF2a2hmRWJvZTdtRGFwTHVBelNra1ZQbWRRQ1dLZ21SS2phK2NNOGlqUmxuaitZVmhnQndOa3ZaM242CkN3Rkx3TVlPQ2pjdGRUZTRGUmxQTHNXYW9XTEJCbyt4SkdjczJFLzIxL2dXWUNRZEFvSUJBUURyaVV5SnpoK2wKc0xrYXBBUzhnckEyaEphTmZJZW5hSUhnQ0UrVmZRRCtEem5Hemh0NGIvLzEwSzlOam9oNTJKTWZPN080KzdIRgpZQVhUQXVBUU5ZWGdFNW56YW1ab0lzbXFFZGR0SDhPL1lLMnoyQXZndVhtcDFES21yUVJHZnpTdTd1UjA2Q1NnCjJoV0RPbG1jMVRuT1E5OGZCeXpFWHBvaDBFcjdKdjJ4K05YTy9KelMyb2ZmUnRpT2JUd3B0emNGRXh3YWFHeTMKNWV4eE9veTBMeTkwR1FxcysvbkpMNGtQS2FYNHlvaEtxQm1DaDRJNElkZWxIWURyWExhTFEzRjhXN0w1dFBCTwpkUzBsTEREZlZTUnZzWEJNU052bzBJbm5TRG5ibEIzZXU4WE4xSzFCcThDR0V2R2JsMDNYMmZKY1M3TXI4TDhICjFmMFk1RCtuakpRZkFvSUJBR1ptb0pSdUNMWlpHaUM0YS9VWXVQQ05UNzdYYk9xUkpTR3J0S2dXZjRwQmpVVzkKeElYYnZOY2FRbExvMVFqdzZqK0xsYlRtVXA0SW1neHZVN0ZVK0o5cHREb0M1WDNvYi9xZlBqa1JLdFg3WWpJWgpESWwvakFMVFRuakQ4VGF5UHgxUDBnZVc0YkJyQkVodlVkblAxREFvdlJWZnd0MU1FbEVkY3NBWUdnUHl5L0hiCmMyUU5lb1k0WFYvMFN0UzdNSlM2a29vLzQvQWFXWFBiR0RHRnlrQjNnc0VyM1MzU1l5UUtKUjFtZy9ITlB2SnMKUUN3eGh3YytLemhER2t2eGFDVy9UalJnVllGQi96d3NNeXZvZFNQSzE3QTZhNW55bXhhWGh2MEt4TlhSTHh5cgovNDNuNm1DN3NDbjU3QXNHOE01VGRYamZzRWswc1U0dFh4ZzhUWkVDZ2dFQkFPZGd6U3Rpclh3YkN2dTh2RnBqCkY2KzJ1Qjd0Z21WV0tKOW9DcVlUVW5KQmlzUmRuR2JIdjI0L1ByYTFtaVA0T1M2cGUzZnc2VVczZW9FN2hkam0KZTBrUWVGL0MyaTdhWGJrUUpjd3hOQjVBWHVNWkRUSUpnaGJ2T1lLNjNFUlVtZkJ2QlVaQzloaDlwRHh4aWE2agpXRkhrSXI5Y1VRSG0zUXJ5ajNZQmE1NzFqelB5dkN2Mk9SdUJnU01JODkwVytwbWxtQ25wQTBQMFBqMU5Bd1BPCmhwTktOazZRaVdTMXFiS1hlMC9YeGVOeEZ2NC9HQkVFVGM4S0dLY2RRTVcxenpvVzZzNmFqS2JHUFQ4VEZqK08KYUdqQXFreUdGbUd5Mm9XMVVnRCtsQWxGOUo2cW1qaTAvRCs1UkZkUVgwdjM4am53czJZcTdiNUkzSVVqVTlUSQpLSWtDZ2dFQkFLeTI1dWtlVWxTWVdQRHl4MWJZSkdtNnZDZXE1TmNMTHVRQi9yMDNBVjR6a0orM2hveGwybk9rCjlQYm1PQlg3RDdwWC96a0ZwQnBoeWM3SFltWVRkSDNtWjYvQkhDZ1ZaRWE5THkwaHU2TFkrdGlzNHR5VmVsV0YKa2pqMGZtd2d4Q3M4MDV2NUlWb1JVeVQzdHlaNFpkTGxjZXExdkIxd3BDekw1OFYweXgwNHBXN1dWd2t2ckdaWQphWmdYK0lEQXpvK3V0LzhwODhTbVJ1Sjd1dWxVN3ljck0wTjNtK1Z6RDlQbks3eE9BdTJEUlUyenhOeXRaTDZhCk0wdTc5aDFweGJpMGFIaHhXMWN3NFFocVNMdkkzRCtKUFk5bkpPVjZrRlVrbWVwTXc4T1FkVkdJVzhBNjZxN2sKWjJDUHJwRFFCMWNPWjlsckZZR1NhU01tM1FCUHkzVT0KLS0tLS1FTkQgUFJJVkFURSBLRVktLS0tLQo=
```

Comment:
shall we generate the certs from cert manager by default same as all other components. If to give the cert directly need to add a parameterized section similar to:
```
            # configs:
            #   tls:
            #     certificatesSecret: "router-cert"
            #     certKeyFilename: ""
            #     certFilename: ""
```
as @tharindu1st has introduced